rotkehlchen/chain/aggregator.py

```diff
@@ -1339,19 +1359,57 @@ def add_accounts_to_all_evm(
 
         Returns a list of tuples of the address and the chain it was added in.
         """
-        filtered_accounts = self.filter_active_evm_addresses(accounts)
-        added_accounts = []
-        for chain, account in filtered_accounts:
-            try:
-                self.modify_blockchain_accounts(
-                    blockchain=chain,
-                    accounts=[account],
-                    append_or_remove='append',
+        added_accounts: list[tuple[SUPPORTED_EVM_CHAINS, ChecksumEvmAddress]] = []
+        # Distinguish between contracts and EOAs
+        for account in accounts:
```

Comment:
in the docstring what is the filter you mention? Also move the comment about contracts and ethereum from where you took this logic